src/Visitor/CodeCoverageMethodIgnoreVisitor.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/**
+ * Copyright © 2017-2018 Maks Rafalko
+ *
+ * License: https://opensource.org/licenses/BSD-3-Clause New BSD License
+ */
+
+declare(strict_types=1);
+
+namespace Infection\Visitor;
+
+use PhpParser\Node;
+use PhpParser\NodeTraverser;
+use PhpParser\NodeVisitorAbstract;
+
+/**
+ * @internal
+ */
+final class CodeCoverageMethodIgnoreVisitor extends NodeVisitorAbstract
+{
+    public function enterNode(Node $node)
+    {
+        if (!$node instanceof Node\Stmt\ClassMethod) {
+            return null;
+        }
+
+        /** @var \ReflectionClass $reflection */
+        $reflection = $node->getAttribute(ReflectionVisitor::REFLECTION_CLASS_KEY);
+
+        $method = $reflection->getMethod($node->name->toString());
+
+        $docComment = $method->getDocComment();
+
+        if ($docComment === false) {
+            return null;
+        }
+
+        if (strpos($docComment, '@codeCoverageIgnore') !== false) {
+            /*
+             * This is a workaround to "disable" this Node from mutation.
```

Comment:
See https://github.com/nikic/PHP-Parser/issues/534 for a feature request to support this case

Reply:
With 4.1.0 released we can bump the minimum version and use that.

Reply:
@BackEndTea done